scripts/container_ut/run-tests.sh

```diff
@@ -1,11 +1,22 @@
 #!/bin/bash -e
+set -o pipefail
 
 [ "${DEBUG,,}" == "true" ] && set -x
 scriptdir=$(realpath $(dirname "$0"))
 
 src_root=/root/contrail
-logs_path="${src_root}/contrail/logs"
+logs_path="${src_root}/logs"
 cd $src_root
 mkdir -p "$logs_path"
+repo_dir="/root/contrail/contrail-container-builder"
 
-/root/contrail/contrail-container-builder/run-tests.sh &> $logs_path/containers-unit-tests.log
+res=0
+targets=$($repo_dir/run-tests.sh list | grep -P '^containers')
```

Comment:
why grep is needed here? should it return just list? or it has some prefix/suffix?
should we check for non-zero exit code?

Reply:
run-tests.sh list the output contains a comment, grep filters it.